### ISSUE TITLE
Tensorboard Learning Curves (Super-imposed train+validation curves) 

### ISSUE THREAD

**Issue by SamuelYvon** [ultralytics/yolov5#6921]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

Hi!

Right now the loss curves for training and validation (as well as COCO-metrics) are split into two different categories (train v.s valid) (at least in tensorboard).

I think it would be better for analysis if the learning curves were super-imposed to clearly show their relation over time. I am not against having them in their respective plots.

### Use case

This is mainly to have a better vision of the learning process (overfitting / underfitting). Right now because the losses are in different plots and the scales might be different, comparing is non-trivial.

### Additional

I am willing to implement this.

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@SamuelYvon I understand what you are saying but we can't overlay multiple metrics on a single plot for a single run, otherwise the framework would not be expandable to multiple runs. i.e. if I do 3 training runs in Tensorboard I see 3 curves in each plot, one per run. This multi-run plotting feature is not compatible with your new feature request.

In general overfitting pertains to the val set, so you can establish whether overfitting is happening just by seeing val losses start to rise.

<img width="1626" alt="Screenshot 2022-03-09 at 21 49 11" src="https://user-images.githubusercontent.com/26833433/157533350-f31d31f7-1e97-43f3-bbec-8b4ee4cff74b.png">

**Comment by SamuelYvon** [comment]
@glenn-jocher 
>I understand what you are saying but we can't overlay multiple metrics on a single plot for a single run, otherwise the framework would not be expandable to multiple runs. i.e. if I do 3 training runs in Tensorboard I see 3 curves in each plot, one per run. This multi-run plotting feature is not compatible with your new feature request.

I see. I haven't used it like that (I mostly look run by run), so it did not appear to be a problem. 

Are you open to have this behind a flag / option?

>In general overfitting pertains to the val set, so you can establish whether overfitting is happening just by seeing val losses start to rise.

This is true, but I prefer looking at how the training loss is behaving as well; both losses can climb in degenerate cases

**Comment by glenn-jocher** [comment]
@SamuelYvon no, I don't think this is line with best practices. Actually I don't think it's even possible with Tensorboard.

**Comment by SamuelYvon** [comment]
Ok, thanks for your time!
